blocks/copd-causes/copd-causes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * adapts content of the COPD Causes block based on the viewport width
+ * @param {Element} block The COPD Causes block element
+ */
+
+let originalBlock = null;
+
+function onResize(block) {
```

Comment:
Please consider this as a proposal, to show the <em> following the layout of the original page.
If we agree on the approach, I can improve  in order to move just the first <em>that opens the paragraph.